projects/aca-content/src/lib/components/sidenav/navigation-menu/navigation-menu.component.html

```diff
@@ -0,0 +1,30 @@
+<button mat-icon-button [matMenuTriggerFor]="menu" aria-label="matMenu">
+    <mat-icon title="{{'APP.TOOLTIPS.OPTIONS_SETTINGS' | translate}}">apps</mat-icon>
+</button>
+<mat-menu #menu="matMenu" xPosition="before">
+    <div mat-menu-item [routerLink]="['/profile']" title="{{'APP.TOOLTIPS.MY_PROFILE' | translate}}">
+        <mat-icon>account_circle</mat-icon>
+        <span>{{ displayName }}</span>
+    </div>
+    <mat-divider></mat-divider>
+    <button mat-menu-item data-automation-id="language-menu-button" [matMenuTriggerFor]="langMenu">
+        <mat-icon>language</mat-icon>
+        <span>{{ 'APP.LANGUAGE' | translate }}</span>
+    </button>
+    <mat-menu #langMenu="matMenu" class="adf-menu">
+        <adf-language-menu></adf-language-menu>
+    </mat-menu>
+    <button mat-menu-item data-automation-id="about" [routerLink]="['/about']">
+        <mat-icon>info</mat-icon>
+        {{ 'APP.BROWSE.ABOUT.TITLE' | translate }}
+    </button>
+    <button mat-menu-item data-automation-id="user-language-menu" [routerLink]="['/settings']">
```

Comment:
copy-pasted code, wrong automation id `data-automation-id="user-language-menu"` for the `settings`